draft-ietf-httpbis-message-signatures.md

```diff
@@ -822,177 +927,16 @@ Content-Length: 18
 {"hello": "world"}
 ~~~
 
-### Signature Generation
```

Comment:
We need to rework all the examples again to use the new syntax and key/alg resolution. For now I've changed parameters and tweaked obvious syntax changes, and removed examples of removed options like hs2019, but we'll need to work through this whole section again.